addons/mail/static/src/core_ui/message.xml

```diff
@@ -92,7 +92,7 @@
                                    }"
                         >
                             <div class="o-mail-Message-content o-min-width-0" t-att-class="{ 'w-100': state.isEditing }">
-                                <div class="o-mail-Message-textContent position-relative d-flex" t-att-class="{ 'w-100': state.isEditing }">
+                                <div class="o-mail-Message-textContent position-relative overflow-x-auto d-flex" t-att-class="{ 'w-100': state.isEditing }">
```

Comment:
If the content overflows, actions are also handled in overflow:
![overflow-actions](https://github.com/odoo/odoo/assets/6569390/6f3b3574-e8bd-43fa-825d-8bbc7b16f99f)
It would be best to have it to just the content:
![overflow-actions-2](https://github.com/odoo/odoo/assets/6569390/2e082741-6722-4298-bbff-0632cc8cf32d)

If you put `overflow-x-auto` in the `position-relative` children, then it matches the latter gif.

Reply:
ok! That's indeed a better solution.